manual/source/Tutorial/GettingStarted/Metafunctions.rst

```diff
@@ -97,10 +120,13 @@ For example the following function prints the length of a fixed sized string usi
 .. includefrags:: demos/tutorial/metafunctions/base.cpp
     :fragment: length
 
-SeqAn Metafunctions
--------------------
+.. includefrags:: demos/tutorial/metafunctions/base.cpp.stdout
+    :fragment: length
+
+.. important::
+      Redundant use of "*Value*":
 
-If you want to search for metafunctions only you can do so by only selecting the metafunction category to the left of the search window at the online documentation.
+      There is a **Type Metafunction** called :dox:`ContainerConcept#Value` (``Value<TSomeType>::Type``), which has nothing to do with the **Value Metafunctions** described above.
 
```

Comment:
first you use capital latters for Type and Value in
> .. returns a Type 

and 
> .. e.g. of Values

but than you switch to lowercase
> .. and not a value.

so just to be picky about consistency i wanted to ask if i might change the latter also to 'Value' ?